lib/node-utils/index.ts

```diff
@@ -374,6 +374,10 @@ export function getPropertyIdentifierNode(
 		return getPropertyIdentifierNode(node.expression);
 	}
 
+	if (ASTUtils.isAwaitExpression(node)) {
+		return getPropertyIdentifierNode(node.argument);
+	}
```

Comment:
`getPropertyIdentifierNode` stopped its recursive iteration when it encountered an await expression, and therefore returned `false`

Reply:
praise: Good catch!